frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -515,25 +518,10 @@ export const funnelLogic = kea<funnelLogicType>({
                 return
             }
             if (searchParams.insight === ViewType.FUNNELS) {
-                const paramsToCheck = {
-                    date_from: searchParams.date_from,
-                    date_to: searchParams.date_to,
-                    actions: searchParams.actions,
-                    events: searchParams.events,
-                    display: searchParams.display,
-                    interval: searchParams.interval,
-                    properties: searchParams.properties,
-                }
-                const _filters = {
-                    date_from: values.filters.date_from,
-                    date_to: values.filters.date_to,
-                    actions: values.filters.actions,
-                    events: values.filters.events,
-                    interval: values.filters.interval,
+                const currentParams = cleanFunnelParams(values.filters, true)
+                const paramsToCheck = cleanFunnelParams(searchParams, true)
```

Comment:
With this change, looks like we're missing sanitization of params. Params that shouldn't be kept aren't discarded.

Reply:
Hmm, they kind of are discarded. That's the `true` bit at the end. It should theoretically only keep params that funnels need. Previously the two objects had differing keys and the funnel was reloaded all the time.